CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Feature Enhancements:
   Implement global `Infinity` property (@AnirudhKonduru)
 - [FEATURE #410](https://github.com/boa-dev/boa/pull/410):
   Add support for the reviver function to JSON.parse (@abhijeetbhagat)
+- [FEATURE #525](https://github.com/boa-dev/boa/pull/525):
+  Implement additional `Math` methods (@mr-rodgers)
```

Comment:
```suggestion
```

`v0.9` has already been released, so this should be left for `v0.10`

Reply:
Hmm, ok!